### DISCUSSION TITLE
Microsoft Teams Integration

### DISCUSSION THREAD

**Discussion by Cinojose** [Netflix/dispatch#3276] [General]
Hi Team 👋 

Wanted to start a thread to see the plans for MS teams integrations similar to the slack plugin we have.

**Comment by kevgliss** [comment]
Hi @Cinojose we don't currently have any plans to integrate MS teams ourselves. That said, we would be able to provide some support/help with the integration if you were considering undertaking it. I will caution that Dispatch's integration with chat (Slack currently) is pretty involved. 

Most of the interfaces involved are abstracted away into the `conversation` plugins. But the amount of code in the slack conversation plugin today is quite extensive. Also, while I imagine there is mostly parity between slack and teams as far as functionality there may be some things that are more awkward to do on one platform vs another. 

Happy to discuss this further, it might also be helpful to scope out an MVP set of features for teams so some limited support could be made available (as opposed to all the chatbot interactions).

**Comment by Cinojose** [comment]
Thanks @kevgliss, I started a thread internally to chat on this feature development. I will keep you posed on the outcome, I wanted to also see the community interest on this feature as the Teams adoption is getting mature.
